thinkcompany/think_dev-standards
# fix: add title to head using SEO component

## Changes

This PR adds a title to the home and landing templates.

_before_:

<img width="517" alt="image" src="https://github.com/user-attachments/assets/3cc1280d-0fa8-4131-afa3-286de9020488">

_after_:

<img width="520" alt="image" src="https://github.com/user-attachments/assets/50ba6941-0a1f-4655-b39f-f9771f4b2c73">


Additionally:
- add an .nvmrc
- add legacy deps
- convert class components to functional components
- replace hero image if/else with switch statement
- remove commented code and console log statements
- replaces dangerouslySetInnerHTML with a utility that sanitizes html string and converts it to JSX

## Checklist
Check all the boxes that apply to this pull request
- [x] Spelling and Formatting have been checked
- [x] Example code has been included where needed
- [x] References and links are included where needed
- [ ] This pull request requires an update to dependencies
- [x] This change looks good on mobile

## How To Review
- Pull this branch, install the dependencies, and run `npm start`
- Verify that each page you visit has a non-empty title element in the `<head>`

## Tickets Closed
List tickets closed by this pull request

- Closes [THINKDOCS-##](#link#)
